docker-compose/docker-compose-injistack/README.md

```diff
@@ -0,0 +1,154 @@
+# Inji Stack Setup
+
+This guide provides instructions for setting up and running Inji Stack.
+
+## Prerequisites
+- Docker and Docker Compose installed on your system
+- Git (to clone the repository)
+- Basic understanding of Docker and container operations
+### Building inji-web-proxy
+Before running the docker-compose, you need to build the inji-web-proxy image:
+
+```bash
+# Clone the repository
+git clone https://github.com/mosip/inji-web.git -b develop
+cd inji-web/inji-web-proxy
+
+# Build the Docker image
+docker build -t inji-web-proxy:local .
+```
+
+## Directory Structure Setup
+Create the following directory structure before proceeding:
+
+```
+docker-compose-injistack/
+├── data/
+│   └── CERTIFY_PKCS12/(p12 file generated at runtime)
+├── certs/
+│   └── oidckeystore.p12 (to be obtained during onboarding of mimoto to esignet)
+├── loader_path/
+│   └── certify/ (plugin jar to be placed here)
+├── config/ (default setup should work as is for csvplugin, any other config changes user can make as per their setup)
+│   ├── certify-default.properties
+│   ├── certify-mock-identity.properties
+│   ├── mimoto-default.properties
+│   ├── mimoto-issuers-config.json
+│   ├── mimoto-trusted-verifiers.json
+│   └── credential-template.html
+├── nginx.conf
+├── certify_init.sql
+└── docker-compose.yml
+```
+
+## Mock Certify Plugin Setup
+You have two options for the certify plugin:
+
+### Option 1: Use Existing Mock Plugin
+- Supported versions: 0.3.0 and above
+- Download the snapshot JAR from:
+  ```
+  https://oss.sonatype.org/content/repositories/snapshots/io/mosip/certify/mock-certify-plugin/0.3.0-SNAPSHOT/
+  ```
+- Place the downloaded JAR in `loader_path/certify/`
+
+### Option 2: Create Custom Plugin
+You can create your own plugin by implementing the following interface and place the resultant jar in loader_path:
+```java
+public interface DataProviderPlugin {
+    // Implement your custom logic here
+}
+```
+
+## Configuration Setup
+
+
+
+### 1. Certificate Setup
+- Place your PKCS12 certificate file (obtained from esignet onboarding) in:
+  ```
+  certs/oidckeystore.p12
+  ```
+
+### 2. Configuration Files
```

Comment:
for all configuration, we should point to respective repo docker-compose readme. By looking at this only, people might not know what is correct configuration.

Reply:
We should also add note about where to use localhost or where to use direct service name in configurations

Reply:
We should also mention about ngrok and any other services to make localhost accessible from mobile or other systems

Reply:
Added Reference to Mimoto Docs, if we make it localhost